bodhi/tests/server/services/test_updates.py

```diff
@@ -2165,24 +2165,34 @@ def test_obsolete_if_unstable_karma_not_reached_with_autopush_enabled_when_pendi
     def test_obsolete_if_unstable_with_autopush_enabled_when_testing(self, publish, *args):
         """
         Send update to obsolete state if it reaches unstable karma threshold on
-        testing state where request is stable when Autopush is enabled. Make sure that it
-        does not go to stable state.
+        testing state where request is stable when Autopush is enabled. Make sure that the
+        autopush is disabled and the update does not go to stable state.
```

Comment:
This comment says that the test should make sure that autopush is disabled, but I don't see an assertion below that autokarma is False.